src/boot/efi/util.h

```diff
@@ -43,6 +58,16 @@ static inline void freep(void *p) {
 
 #define _cleanup_free_ _cleanup_(freep)
 
+static __always_inline void erase_obj(void *p) {
+        size_t l;
+        assert_cl(p != NULL);
```

Comment:
in our coding style, we'd write that as `assert_cl(p)`, i.e. we rely on C's downgrade-to-bool feature for pointer NULL checks

Reply:
Ack.